### ISSUE TITLE
Select child nodes when select parent node.

### ISSUE THREAD

**Issue by jeevasusej** [minop1205/react-dnd-treeview#141]
When do check the parent node checkbox, select parent node's children's checkbox by default and vice versa too.

**Comment by minop1205** [comment]
@jeevasusej Is it a problem with the following demo?
https://minop1205.github.io/react-dnd-treeview/?path=/docs/examples-tree-multiple-selections--multiple-selections-story

Is there any way to confirm the issues you mentioned?

**Comment by jeevasusej** [comment]
Yes, when we clicked on the child (checkbox), we do not know whether the parent is selected or not.

**Comment by nilesh2b** [comment]
@jeevasusej Yes right.

**Comment by nilesh2b** [comment]
@minop1205 Any solution here?

**Comment by slince** [comment]
+1

**Comment by nilesh2b** [comment]
@jeevasusej Did you find any solution for the same?

**Comment by ebenezar** [comment]
I have written a wrapper and used the following jquery fancytree, which solves all my problems. 

https://github.com/mar10/fancytree
